tunnel/intercept/tproxy/tproxy_linux.go

```diff
@@ -106,6 +106,7 @@ func (self *interceptor) Stop() {
 		proxy.Stop(alwaysRemoveAddressTracker{})
```

Comment:
We were removing the NF-INTERCEPT chain whenever we removed any service, which caused all service intercepts to get cleaned up.